src/Tooltip/index.js

```diff
@@ -205,6 +215,17 @@ class Tooltip extends React.PureComponent<Props, State> {
     align: ALIGNS.CENTER,
     shown: false,
     shownMobile: false,
+    positions: {
```

Comment:
It was really necessary to save positions in the state? the method `getDimensions` is called every time it needs to calculate it.

Reply:
I'm open to discussion, I was thinking about just a boolean state variable, which would force reRender in similar way, but that seem pointless because positions are changed in the static anyway and adding state that doesn't really do anything seemed strange.But it might be more performant. What do you think, do you have any other solutions in mind?